examples/dgmulti_2d/elixir_navierstokes_convergence.jl

```diff
@@ -10,180 +10,192 @@ mu() = 0.01
+    # see also https://github.com/trixi-framework/Trixi.jl/pull/1160
+    inv_gamma_minus_one = inv(equations.gamma - 1)
+    Pr = prandtl_number()
+    mu_ = mu()
+
+    # Same settings as in `initial_condition`
+    # Amplitude and shift
+    A = 0.5
+    c = 2.0
+
+    # convenience values for trig. functions
+    pi_x = pi * x[1]
+    pi_y = pi * x[2]
+    pi_t = pi * t
+
+    # compute the manufactured solution and all necessary derivatives
+    rho = c + A * sin(pi_x) * cos(pi_y) * cos(pi_t)
+    rho_t = -pi * A * sin(pi_x) * cos(pi_y) * sin(pi_t)
+    rho_x = pi * A * cos(pi_x) * cos(pi_y) * cos(pi_t)
+    rho_y = -pi * A * sin(pi_x) * sin(pi_y) * cos(pi_t)
+    rho_xx = -pi * pi * A * sin(pi_x) * cos(pi_y) * cos(pi_t)
+    rho_yy = -pi * pi * A * sin(pi_x) * cos(pi_y) * cos(pi_t)
+
+    v1 = sin(pi_x) * log(y + 2.0) * (1.0 - exp(-A * (y - 1.0))) * cos(pi_t)
+    v1_t = -pi * sin(pi_x) * log(y + 2.0) * (1.0 - exp(-A * (y - 1.0))) * sin(pi_t)
+    v1_x = pi * cos(pi_x) * log(y + 2.0) * (1.0 - exp(-A * (y - 1.0))) * cos(pi_t)
+    v1_y = sin(pi_x) *
+           (A * log(y + 2.0) * exp(-A * (y - 1.0)) +
+            (1.0 - exp(-A * (y - 1.0))) / (y + 2.0)) * cos(pi_t)
+    v1_xx = -pi * pi * sin(pi_x) * log(y + 2.0) * (1.0 - exp(-A * (y - 1.0))) * cos(pi_t)
+    v1_xy = pi * cos(pi_x) *
+            (A * log(y + 2.0) * exp(-A * (y - 1.0)) +
+             (1.0 - exp(-A * (y - 1.0))) / (y + 2.0)) * cos(pi_t)
+    v1_yy = (sin(pi_x) *
+             (2.0 * A * exp(-A * (y - 1.0)) / (y + 2.0) -
+              A * A * log(y + 2.0) * exp(-A * (y - 1.0)) -
+              (1.0 - exp(-A * (y - 1.0))) / ((y + 2.0) * (y + 2.0))) * cos(pi_t))
+    v2 = v1
+    v2_t = v1_t
+    v2_x = v1_x
+    v2_y = v1_y
+    v2_xx = v1_xx
+    v2_xy = v1_xy
+    v2_yy = v1_yy
+
+    p = rho * rho
+    p_t = 2.0 * rho * rho_t
+    p_x = 2.0 * rho * rho_x
+    p_y = 2.0 * rho * rho_y
+    p_xx = 2.0 * rho * rho_xx + 2.0 * rho_x * rho_x
+    p_yy = 2.0 * rho * rho_yy + 2.0 * rho_y * rho_y
+
+    # Note this simplifies slightly because the ansatz assumes that v1 = v2
+    E = p * inv_gamma_minus_one + 0.5 * rho * (v1^2 + v2^2)
+    E_t = p_t * inv_gamma_minus_one + rho_t * v1^2 + 2.0 * rho * v1 * v1_t
+    E_x = p_x * inv_gamma_minus_one + rho_x * v1^2 + 2.0 * rho * v1 * v1_x
+    E_y = p_y * inv_gamma_minus_one + rho_y * v1^2 + 2.0 * rho * v1 * v1_y
+
+    # Some convenience constants
+    T_const = equations.gamma * inv_gamma_minus_one / Pr
+    inv_rho_cubed = 1.0 / (rho^3)
+
+    # compute the source terms
+    # density equation
+    du1 = rho_t + rho_x * v1 + rho * v1_x + rho_y * v2 + rho * v2_y
+
+    # x-momentum equation
+    du2 = (rho_t * v1 + rho * v1_t + p_x + rho_x * v1^2
+           + 2.0 * rho * v1 * v1_x
+           + rho_y * v1 * v2
+           + rho * v1_y * v2
+           + rho * v1 * v2_y -
+           # stress tensor from x-direction
+           4.0 / 3.0 * v1_xx * mu_ +
+           2.0 / 3.0 * v2_xy * mu_ -
```

Comment:
It is still a bit weird that sometimes the binary operator is in the beginning of the line and sometimes at the end, but I could live with that.